portality/templates/layouts/static_page.html

```diff
@@ -1,9 +1,23 @@
 {% extends "layouts/base.html" %}
+{% set page = data.frontmatter[page_frag] %}
 
-{% block page_title %}{{ page_title }} &ndash; {% endblock %}
+{% block page_title %}{{ page.title }} &ndash; {% endblock %}
```

Comment:
Should check what's happening with page titles on merge due to https://github.com/DOAJ/doaj/pull/1875

Reply:
Good spot, have carried that forward during the merge